jira7/Package.md

```diff
@@ -1,72 +1,94 @@
-   
-   *Note -JIRA’s REST API is protected by the same restrictions which are provided via JIRAs standard web interface.
-    This means that if you do not have valid jira credentials, you are accessing JIRA anonymously. Furthermore, 
-    if you log in and do not have permission to view something in JIRA, you will not be able to view it using the 
-    Ballerina JIRA Connector as well.*
+The `wso2/jira` package contains operations that get all details of a given project component, delete existing 
+components, create a new project component related to a specific project, etc.
 
-3. Create a new Ballerina project by executing the following command.
-      ```shell
-        <PROJECT_ROOT_DIRECTORY>$ ballerina init`
-      ```
-  
-4. Import the jira package to your Ballerina program as follows.This will download the jira7 artifacts from the 
-`ballerina central` to your local repository.
+**JIRA Issue Operations**
+
+The `wso2/jira` package contains operations that get all the details of a given issue using the issue key, delete existing 
+issues, create new issues, etc.
+
+## Compatibility
+|                    |    Version     |  
+| :-----------------:|:--------------:| 
+| Ballerina Language | 0.970.0-beta15 |
+| JIRA REST API      |    7.2.2       |  
 
-    ```ballerina
-       import wso2/jira7;
-    ```
+## Sample
+The JIRA connector can be used to get summaries of all projects, create a project, and create an issue. First, import 
+the `wso2/jira7` package into the Ballerina project.
+```ballerina
+import wso2/jira7;
+```
+**Obtaining Credentials to Run the Sample**
 
-5. Provide the credentials to your endpoint in the initialization step, and use as shown 
-in the following sample code.
+1. Visit [Atlassian](https://id.atlassian.com/signup) and create an Atlassian Account.
+2. Obtain the following credentials from the Twilio dashboard:
+    * Username
+    * Password  
 
-    ```ballerina
-       import wso2/jira7 as jira;
-       
-       function main(string... args) { 
-         
-           //Creating the jira endpoint
-           endpoint jira:Client jiraEndpoint {
-                
-                httpClientConfig:{
-                    url:"https://support-staging.wso2.com/jira",
-                    auth:{
-                        scheme:"basic",
-                        username:"username",
-                        password:"password"
-                    }
-                }
-           };
-           
-           jira:JiraConnectorError jiraError = {};
-           jira:Project project = {};
-           string projectKey = "RRDEVSPRT";    
-           //Endpoint Action
-           var result = jiraEndpoint -> getProject(projectKey);
-           match result{
-               jira:Project p => project = p;
-               jira:JiraConnectorError e => jiraError = err;
-           }
-       }
-        
-    ```
\ No newline at end of file
+You can now enter the credentials and base URL in the HTTP client config.
```

Comment:
The base URL is specified outside of the HTTP client config. Is this text or the sample incorrect?

Reply:
Modified the code to get the base URL inside the HTTP client config. Updated the sample.